.github/actions/code-coverage/action.yml

```diff
@@ -0,0 +1,77 @@
+name: Code Coverage Report
+description: 'Adds a comment to PRs with code coverage'
+
+inputs:
+  coverage-counter-type:
+    description: 'Report counter type used to calculate coverage metrics. Possible values are: INSTRUCTION, LINE or BRANCH.'
+    required: true
+    default: 'LINE'
+  gradle_command:
+    description: 'The gradle command to execute to generate the code coverage report'
+    required: true
+    default: 'koverXmlReport'
```

Comment:
Do defaults and required together make sense?

Reply:
It made sense at the time 😅 . Thought was that having required set to true even with a default given would not allow a null or empty value to be supplied. I went ahead and removed required wherever a default is given.